------------------------------------------------------------
revno: 258
committer: Scott Moser <smoser@ubuntu.com>
branch nick: trunk
timestamp: Wed 2010-09-08 10:17:19 -0400
message:
  remove broken (syntax) try/catch in UserDataHandler.py
=== modified file 'cloudinit/UserDataHandler.py'
--- a/cloudinit/UserDataHandler.py
+++ b/cloudinit/UserDataHandler.py
@@ -46,9 +46,7 @@
     for line in str.splitlines():
         if line == "#include": continue
         if line.startswith("#"): continue
-        try:
-            content = urllib.urlopen(line).read()
-        except Exception e:
+        content = urllib.urlopen(line).read()
         process_includes(email.message_from_string(decomp_str(content)),parts)
 
 def process_includes(msg,parts):
